src/libraries/Microsoft.Bcl.Numerics/src/System/MathF.cs

```diff
@@ -0,0 +1,318 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*============================================================
+**
+** Purpose: Some single-precision floating-point math operations
+**
+===========================================================*/
+
+//This class contains only static members and doesn't require serialization.
```

Comment:
nit: It would be beneficial to have a comment at the top here elaborating that this simply defers to `System.Math` and why it does so on .NET Framework.

It might also be nice to call out the few places that explicitly differ in handling from .NET Core (such as around `-0.0f`)

Reply:
Good idea. Added that in.